Backend/app/rider-platform/rider-app/Main/src/Tools/Notifications.hs

```diff
@@ -562,3 +562,30 @@ notifySafetyAlert booking reason = do
           [ reason
           ]
   notifyPerson person.merchantId person.merchantOperatingCityId notificationData
+
+notifyDriverBirthDay ::
+  ServiceFlow m r =>
+  Id Person ->
+  Text ->
+  m ()
+notifyDriverBirthDay personId driverName = do
+  person <- Person.findById personId >>= fromMaybeM (PersonNotFound personId.getId)
+  let merchantOperatingCityId = person.merchantOperatingCityId
+  let notificationData =
+        Notification.NotificationReq
+          { category = Notification.DRIVER_BIRTHDAY,
+            subCategory = Nothing,
+            showNotification = Notification.SHOW,
+            messagePriority = Nothing,
```

Comment:
why is it Nothing ?

Reply:
cz it has no priority , can be sent  or can't be sent depends on the dob exists or not